src/main/java/com/openshift/restclient/model/IReplicationController.java

```diff
@@ -70,7 +71,7 @@
 	 * @param containerPorts  the container ports
 	 * @param volumes		 the set of emptyDir volumes to add to the config
 	 */
-	void addContainer(String name, DockerImageURI tag,  Set<IPort> containerPorts, Map<String, String> envVars, List<String> volumes);
+	IContainer addContainer(String name, DockerImageURI tag,  Set<IPort> containerPorts, Map<String, String> envVars, List<String> volumes);
```

Comment:
It doesn't break consumers. It breaks potential implementers. The rest client is the obvious only implementation, but that might break someone's unit tests that mocked that interface.